Libraries/LibJS/AST.cpp

```diff
@@ -120,6 +172,24 @@ Value BinaryExpression::execute(Interpreter& interpreter) const
         return sub(lhs_result, rhs_result);
     case BinaryOp::TypedEquals:
         return typed_eq(lhs_result, rhs_result);
+    case BinaryOp::TypedInequals:
+        return Value(!typed_eq(lhs_result, rhs_result).as_bool());
+    case BinaryOp::Greater:
+        return greater(lhs_result, rhs_result);
+    case BinaryOp::Smaller:
+        return Value(!greater(lhs_result, rhs_result).as_bool());
```

Comment:
Careful here. The negation of `>` is `<=` not `<`.

Reply:
Oops, that's an embarrassing error on my end :sweat_smile: 

Anyways, fixed!